ext/RMagick/rmilist.c

```diff
@@ -359,6 +359,7 @@ ImageList_flatten_images(VALUE self)
  * @param argv array of input arguments
  * @param self this object
  * @return a new image
+ * @deprecated This method has been deprecated. Please use Image_fx.
```

Comment:
Does this cause a deprecation warning to  be output when a user calls the method? Or do we need to add a warning to method, too?

Reply:
Good point, adding a warning will probably be better for our users.

p.s. I also noticed some other deprecated methods. Maybe these can be remove in the 4.0 release?

Reply:
Yeah, I think that would make sense, though we should make sure users have warnings before then. I'm hoping to get a 3.1 release out in the next week or so, btw, though I'd imagine we'll probably have a 3.2 and maybe a 3.3 before we get to 4.0, so we've got some time to get in the deprecations we want.

Reply:
Okay I will open a bunch of PR's then 😏